src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedProjectReference.xaml

```diff
@@ -25,6 +25,17 @@
                   SourceOfDefaultValue="AfterContext" />
     </StringListProperty.DataSource>
   </StringListProperty>
+  
+  <StringProperty Name="BrowsePath"
+                  ReadOnly="True"
+                  Visible="False">
+    <StringProperty.DataSource>
+      <DataSource ItemType="ProjectReference"
+                  PersistedName="Identity"
+                  Persistence="Intrinsic"
```

Comment:
>Intrinsic [](start = 31, length = 9)

What determines whether this is Intrinsic vs whether the attribute is excluded?

Reply:
They are two different attributes; `Persistence` determines what backing "object" reads the value from the item and `PersistenceStyle` determines whether its rewritten out as an attribute or an element.